### PR TITLE
Refactoring using classes

### PR DESCRIPTION
**Pull request by arabiu033**
- Modified the javascript code to work with classes
- Added some styles to the design
- Fix linters

Files Modified

- index.html
- style.css
- script.js

### CODE REVIEW

**Review by DammyShittu — COMMENTED** [review]
Hi @arabiu033,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### You do not have a descriptive PR summary

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: script.js [hunk 1]
```diff
@@ -2,54 +2,63 @@ const bookTitle = document.getElementById('book-title');
...
-  displayBooks();
+  books = new BookHandling(JSON.parse(localStorage.getItem('collectionOfBooks') || '[]'));
 }
 
 addBtn.addEventListener('click', () => {
-  bookCount += 1;
-  addBook(bookTitle.value, bookAuthor.value);
-  collectionOfBooks.push({ id: bookCount, title: bookTitle.value, author: bookAuthor.value });
-  localStorage.setItem('collectionOfBooks', JSON.stringify(collectionOfBooks));
+  books.bookCount += 1;
```
Comment (smunozmo) [inline script.js:53]: - [ ] Excellent job refactoring your code! However, it's showing an error when I try to add a book:

Screen Shot 2022-09-20 at 2 59 04 PM

Therefore, the book is not added so I was not able to test the app, I would suggest fixing this bug in order to be able to test it.

**Review by smunozmo — CHANGES_REQUESTED** [review]
Hi @arabiu033, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (arabiu033) [reply script.js:53]: Debugging this will be very challenging since I'm not experiencing it from my side. Alright thanks, let me look into it.

Comment (arabiu033) [reply script.js]: Done. Handle it.

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @arabiu033 👋, 

Well done for implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [x] Kindly add the "Getting Started" section to your README. Please use this [template](https://github.com/microverseinc/readme-template) as a reference
- [x] This is a collaborative project but you haven't collaborated with your coding partner as I cannot see any of their commits in this project. Please endeavour to always work with your coding partners in the future 🙏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**